build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.0.0-beta4'
```

Comment:
I think this is causing the travis build to fail, probably coverage output has a different directory now. Let's revert this and I will update it later :)

Reply:
My mistake :(